internal/db/resource.go

```diff
@@ -106,3 +104,68 @@ func (d *Database) ListResources(ctx context.Context, limit, offset int) (_ []Re
 	}
 	return resources, nil
 }
+
+// buildQuery is an utility function to build the database query according to two optional parameters.
+// nameFilter: used to match resources name.
+// typeFilter used to match resources type. If this is not empty the resources are fetched from a single table.
```

Comment:
For consistency with the previous line.
```suggestion
// typeFilter: used to match resources type. If this is not empty the resources are fetched from a single table.
```